easybuild/easyconfigs/l/libmatheval/libmatheval-1.1.11-foss-2016b.eb

```diff
@@ -0,0 +1,33 @@
+easyblock = 'ConfigureMake'
+
+name = 'libmatheval'
+version = '1.1.11'
+
+homepage = 'http://www.gnu.org/software/libmatheval/'
+description = """GNU libmatheval is a library (callable from C and Fortran) to parse
+ and evaluate symbolic expressions input as text."""
+
+toolchain = {'name': 'foss', 'version': '2016b'}
+toolchainopts = {'pic': True}
+
+sources = [SOURCELOWER_TAR_GZ]
+source_urls = [GNU_SOURCE]
+
+dependencies = [
+    ('flex', '2.6.0'),
+    ('Bison', '3.0.4'),
+    ('byacc', '20160606'),
```

Comment:
`byacc` only provides `bin/yacc`, while Bison also provides `bin/bison` and `lib/liby.a`...

Not sure if we can drop either. @wpoely86 any ideas?